dts/bindings/usb/raspberrypi,pico-usbd.yaml

```diff
@@ -1,11 +1,32 @@
 # Copyright (c) 2021, Pete Johanson
 # SPDX-License-Identifier: Apache-2.0
 
-description: RaspberryPi Pico USB Device Controller
+description: |
+  RaspberryPi Pico USB Device Controller.
+
+  When the rp2040 is operated as a self powered device,
+  VBUS detection needs to be enabled to ensure the correct
+  pull up handling on the data lines.
+  Example of enabling the controller with vbus detection:
```

Comment:
The VBUS detection is not really tied to self-powered or not, and the device could be connected to a dedicated charging port where function of the device controller is not required. So please change the description to a simple text how to describe the USB device controller on RP2040 in the devicetree to enable it. Also please fix the commit message.